### DISCUSSION TITLE
Latitudinal lines in 0.25deg analysis (CFSR)

### DISCUSSION THREAD

**Discussion by liuxiao37k** [NOAA-EMC/godas#364] [General]
**Description of Issue**: 
Latitudinal lines in 0.25deg analysis (see figure below). Visually detectable after one 24-cycle in almost all variables. Started to be seen in northern high latitudes before propagating into the low latitudes. Pattern strengthens while cycling.

<img width="468" alt="image" src="https://user-images.githubusercontent.com/2281138/181161232-d0bde5ca-fff6-41f7-bf29-533882c54e01.png">
<img width="194" alt="image" src="https://user-images.githubusercontent.com/2281138/181161255-439553ef-5896-4576-a176-4ff0abbe8d9b.png">

Both UFS model and soca-science repo cloned and built on Jun 28, develop branch.

I ran the following tests to debug:
a) turned off DA, still problematic, indicating a model related issue.
b) noda mode, turned off BGC, still problematic, indicating it has nothing to do with BGC.
c) noda mode, no BGC, switched to ufs p7c and older version of ufs_configs, still problematic, so it has to be initial conditions or forcing!
checked restart files - all look good. Replaced with a new set of restart files, still problematic...
d) Switched to GEFS, problem solved!
e) Switched back to CFSR, problem reappeared!
f) Kept CFSR, turned on bias correction, problem solved!

I am glad that problem is solved (by using GEFS, or CFSR w/ bias correction on), but I am surprised that 1) forcing could cause issues like this, and 2) bias correction is strong enough to correct this!

I still need to understand what was exactly the problem (e.g. which forcing) but I'll sleep on it.

**Comment by liuxiao37k** [comment]
CFSR forcing and bias correction package used on Orion: 
/work/noaa/ng-godas/marineda/DATM_INPUT

Orion is down for 2 days, so I made a copy on Gaea:
/lustre/f2/dev/ncep/Xiao.Liu/forcing/DATM_INPUT

**Reply by guillaumevernieres** [reply]
Thanks for documenting the issue and providing a fix @liuxiao37k .
Is the bias correction doing anything else but removing the bias from the forcing? I'm curious-ish now ...

**Reply by liuxiao37k** [reply]
> Thanks for documenting the issue and providing a fix @liuxiao37k .
> Is the bias correction doing anything else but removing the bias from the forcing? I'm curious-ish now ...

@guillaumevernieres That's why I planned to bug YC today! I am curious, and suspicious...

**Comment by travissluka** [comment]
FYI, I do remember seeing these lovely lines in CFSR when I generated those bias correction files... not much to be done about it I think

**Reply by liuxiao37k** [reply]
> FYI, I do remember seeing these lovely lines in CFSR when I generated those bias correction files... not much to be done about it I think

Thank you, @travissluka. A relief to know that it's not my mistake that caused those lines. Do you recall which forcing variables look more suspicious? I looked a few of them and couldn't tell, but I did notice stronger line pattern in the "mixed layer irradiance" field from the BGC module. I buried my head and switched to GEFS to move on.

**Reply by travissluka** [reply]
Its been so long ago, my memory is a bit fuzzy... but I do think I remember seeing a hint of the line patterns in the downward radiation fields. But, when MOM6 interpolates the input atmospheric fields to the tripolar grid, something about that really amplifies the weird pattern, so I think I actually saw it the most with the radiation field diagnostic as reported by MOM6.

**Reply by liuxiao37k** [reply]
Thank you!!

**Reply by hyunchul386** [reply]
@liuxiao37k, sorry late reply. Would you point me the what time the issue occurred in CFSR?

**Comment by liuxiao37k** [comment]
@hyunchul386 thank you! I noticed these lines for two testing periods, one starting Jan 2019, and one starting Apr 2019, although I suspect it applies to other earlier periods too (as Travis pointed out).

**Comment by hyunchul386** [comment]
![CFSR_GEFS_2019011512](https://user-images.githubusercontent.com/14810515/183157706-9ee15f6f-71dc-45a5-b6fe-be568a29f334.png)
![CFSR_GEFS_2019041512](https://user-images.githubusercontent.com/14810515/183157707-be8748ef-0338-499a-9fef-2981351a431a.png)

**Reply by hyunchul386** [reply]
Just two checks for snap shots in 20190115 12Z and 20190415 12Z between CFSR and GEFS. 
It looks there are some peaks in the northern high latitude in CFSR and need to look at the high latitude region in CFSR.
